commands.js

```diff
@@ -1779,6 +1786,7 @@ var commands = exports.commands = {
 		}
 		return;
 	},
+	memoryusagehelp: ["/memoryusage - Shows the amount of memory the server is using. Requires: Console Access"],
```

Comment:
Does this command even do anything? iirc, the last time I tried it, it froze PS. Maybe we should just remove it.